com.unity.render-pipelines.core/Editor/Volume/VolumeComponentEditor.cs

```diff
@@ -176,9 +196,18 @@ internal void Init(VolumeComponent target, Editor inspector)
             serializedObject = new SerializedObject(target);
             activeProperty = serializedObject.FindProperty("active");
             m_AdvancedMode = serializedObject.FindProperty("m_AdvancedMode");
+
+            InitParameters();
+
             OnEnable();
         }
 
+        void InitParameters()
+        {
+            m_VolumeNotAdditionalParameters = new List<VolumeParameter>();
+            VolumeComponent.FindParameters(target, m_VolumeNotAdditionalParameters, field => field.GetCustomAttribute<AdditionalAttribute>() == null);
```

Comment:
@JulienIgnace-Unity Shouldn't you use this attribute too for your PR on the displays of them? Can be perhaps usefull if we have a equivalent to default inspector for volumes.

Reply:
We were discussing this, an we will look at the merge.